soc/nordic/nrf54h/bicr/bicrgen-schema.json

```
@@ -0,0 +1,422 @@
+{
+  "title": "nRF54H20 BICR Configuration",
+  "type": "object",
+  "properties": {
+    "power": {
+      "type": "object",
+      "title": "Power supply configuration",
+      "properties": {
+        "scheme": {
+          "type": "string",
+          "title": "Power supply scheme",
+          "enumNames": [
+            "Unconfigured, system will not boot",
+            "VDDH supplied with 2.1-5.5V and VDD regulated by the chip, inductor present",
+            "Both VDD and VDDH supplied with 1.8V, inductor present"
```

These are separate statements the way I read it.
```suggestion
            "VDDH supplied with 2.1-5.5V and VDD regulated by the chip; inductor present",
            "Both VDD and VDDH supplied with 1.8V; inductor present"
```